### PR TITLE
Update the applying security to a REST API page

### PR DESCRIPTION
**Pull request by gabilang**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/develop/advanced-development/applying-security-to-an-api.md [hunk 1]
```diff
@@ -68,18 +70,18 @@ public class BasicAuthHandler implements Handler {
                 }
             }
         }
-        return false;
+        return true;
```
Comment (chanikag) [inline en/docs/develop/advanced-development/applying-security-to-an-api.md:73]: Why is this changed? The default return value should be false.

Comment (gabilang) [reply en/docs/develop/advanced-development/applying-security-to-an-api.md]: Yeah, it was a mistake while copy. Corrected.

File: en/docs/develop/advanced-development/applying-security-to-an-api.md [hunk 2]
```diff
@@ -128,38 +130,41 @@ and restart the Micro Integrator.
...
+                        </m0:getQuote>
+                    </format>
+                    <args>
+                        <arg evaluator="xml" expression="get-property('uri.var.symbol')"/>
+                    </args>
+                </payloadFactory>
+                <header name="Action" scope="default" value="urn:getQuote"/>
+                <send>
+                    <endpoint key="SimpleStockQuoteEndpoint"/>
+                </send>
```
Comment (chanikag) [inline en/docs/develop/advanced-development/applying-security-to-an-api.md:153]: Shouldn't there a respond mediator here? Is this working with this?

Comment (chanikag) [reply en/docs/develop/advanced-development/applying-security-to-an-api.md:153]: Let's replace this with a call mediator

Comment (gabilang) [reply en/docs/develop/advanced-development/applying-security-to-an-api.md:153]: > Shouldn't there a respond mediator here? Is this working with this?

Yes, it secures the API by requiring user name and password. 
I'll replace this with `call`

Comment (gabilang) [reply en/docs/develop/advanced-development/applying-security-to-an-api.md]: Addressed with https://github.com/wso2/docs-mi/pull/371/commits/d248c358e25efa7125c72ec949f8b0e66730c6be

**Review by chanikag — APPROVED** [review]
